src/account/templates/base.html

```diff
@@ -74,9 +74,9 @@
   <div class="navbar navbar-default navbar-static-bottom text-center">
   <div class="navbar-footer">
     <a href="/www/" class="btn navbar-btn">About</a> &nbsp; &nbsp;
-    <a href="{% static 'docs/html/index.html' %}" class="btn navbar-btn">API Docs</a> &nbsp; &nbsp;
+    <a href="{% static 'docs/html/index.html' %}" class="btn navbar-btn">Documentation</a> &nbsp; &nbsp;
     <a href="https://github.com/datahuborg/datahub" class="btn navbar-btn">GitHub Repo</a> &nbsp; &nbsp;
-    <a href="https://github.com/datahuborg/datahub/tree/master/src/examples" class="btn navbar-btn">Sample Code</a>
+    <a href="/api-docs" class="btn navbar-btn">API</a>
```

Comment:
Instead of hardcoding /api-docs, do a url lookup with {% url %} so it's easier to change the URL in the future.